frontend/Login/SignUp.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { View, StyleSheet, Button, Text } from "react-native";
+import { Feather, FontAwesome5 } from "@expo/vector-icons";
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+import CleanInput from "./CleanInput";
+import Header from "../components/Header";
+import { RootStackParamList } from "../route-settings";
+
+const styles = StyleSheet.create({
+  wrapper: {
+    display: "flex",
+    alignItems: "center",
+  },
+  icon: {
+    paddingTop: 100,
+    paddingBottom: 150,
+  },
+  error: {
+    color: "red",
+    textAlign: "left",
+    width: "70%",
+    fontSize: 11,
+    fontWeight: "bold",
+    marginTop: -11,
+  },
+});
+
+type SignUpProps = NativeStackScreenProps<RootStackParamList, "CreateAccount">;
+
+function SignUp({ navigation, route }: SignUpProps) {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const { isMBO } = route.params;
+
+  const [passwordError, setPasswordError] = useState("");
+
+  const submit = () => {
+    if (password === confirmPassword) {
+      setPasswordError("");
+      console.log({ name, email, password });
```

Comment:
Is this console log for debugging purposes?

Reply:
Yeah, its a placeholder for when we connect it to the backend.